app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaViewModel.kt

```diff
@@ -319,12 +319,13 @@ class ImportMediaViewModel @Inject constructor(
 
     private suspend fun handleImportedAsset(
         context: Context,
-        mimeType: String,
+        importedAssetMimeType: String,
         uri: Uri
     ): ImportedMediaAsset? = withContext(dispatchers.io()) {
         val assetKey = UUID.randomUUID().toString()
         val fileMetadata = uri.getMetaDataFromUri(context)
         val tempAssetPath = kaliumFileSystem.tempFilePath(assetKey)
+        val mimeType = fileMetadata.mimeType.ifEmpty { importedAssetMimeType }
```

Comment:
Check first mimeType from Uri, if not empty, then take the less accurate one from Intent